src/pages/index.js

```diff
@@ -88,19 +130,37 @@ const addProfileFormValidator = new FormValidator(
 
 addProfileFormValidator.enableValidation();
 
+const changeProfilePictureFormValidator = new FormValidator(
+  validationConfig,
+  profilePictureModal
+);
+
+changeProfilePictureFormValidator.enableValidation();
+
 /* -------------------------------- USER INFO ------------------------------- */
 const userProfile = new UserInfo(profileTitle, profileDescription);
 
 /* --------------------------------- #MODALS# --------------------------------- */
+function ButtonLoading() {
+  submitButton.textContent = "Loading...";
+  setTimeout(() => {
```

Comment:
`setTimeout` is not needed because you need to 100% wait for the server response